docs/src/snforge-advanced-features/fork-testing.md

```diff
@@ -75,6 +75,13 @@ From this moment forks can be set using their name in the `fork` attribute.
 {{#include ../../listings/snforge_advanced_features/crates/fork_testing/tests/name.cairo}}
 ```
 
+In some cases you may want to override `block_id` defined in the `Scarb.toml` file.
+You can do it by passing `block_number`, `block_hash`, `block_tag` arguments to the `fork` attribute.
+
+```rust
+#[fork("SEPOLIA", block_number: 654321)]
+```
```

Comment:
I'd make it a whole snippet and `#include` to make sure it stays up to date in the future